src/api/mapping.ts

```diff
@@ -45,6 +45,13 @@ mappingRouter.post('/', (req, res) => {
   const scriptPath = '.scripts'
 
   const respond = () => {
+    const verifySubDomain = domainKeys.find(
```

Comment:
this check should not be in the `respond` function, it might cause errors in the future. in line 19 there's already a check. Just move this to around the same area, so all the checks stay around the same area.